### DISCUSSION TITLE
Translating Pinia Docs with Crowdin project.

### DISCUSSION THREAD

**Discussion by otabekoff** [vuejs/pinia#1910] [Ideas]
Hey, what about using [Crowdin project](https://crowdin.com) for translation purposes? I think it is more faster way. Also, pinia website is small. So it would be easy. There already [Vue](https://crowdin.com/project/vuejs), [Vuetify](https://crowdin.com/project/vuetify) and [Vuesax](https://crowdin.com/project/vuesax) using it.

**Comment by posva** [comment]
See https://github.com/vuejs-translations/guidelines/blob/main/README.md#syncing-with-upstream

**Comment by comradekingu** [comment]
Read the terms and conditions of Crowdin, it is really not something to subject translators to in terms of
usability or signing away shared rights.
Replicating the success of "last activity 7 months-", and "3 years ago" can't be good.
Markdown isn't a strong suit of https://docs.weblate.org/en/latest/formats.html
, but at least it is a functional tool.
